### DISCUSSION TITLE
Affiliate reports in OpenCart 4?

### DISCUSSION THREAD

**Discussion by JoshuaAGE** [opencart/opencart#12143] [General]
OpenCart 4 Demo: https://demo.opencart.com/admin/index.php?route=report/report

OpenCart doc: http://docs.opencart.com/en-gb/report/marketing/commission/

The OpenCart doc describe affiliate reports but at the demo they cannot be found.

Is this a bug? Or is this active after installing?

**Comment by felipoantonoff** [comment]
I think they took away part of the Affiliates feature, I think it started in Opencart 3 and it should be similar in 4, but I'm not sure as I haven't used the Affiliate feature for years.
